### PR TITLE
Post on Good Personhood

### PR DESCRIPTION
**Pull request by benhamill**
Adds a post on whether it matters if someone is a "good person" or not.

Also fixes some typos and clarifies the old SJ is Empathy post.

### CODE REVIEW

File: source/_posts/2016-11-21-are-you-a-good-person.markdown [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+
+## What About that "Good Person" Thing?
+
+So, you make a remark and someone tells you it contained ableist language. You
+didn't know! You appologize and tell the person it was an accident. What's going
+on here? You're assuring them that even though you messed up, you're a good
+person on the inside. But this is treating goodness as an identity. Basically no
+one thinks they're a bad person. You don't be a bad person by thinking it and
+revelling in your badness. That's some Sekeletor-level bullshit and adults
```
Comment (benhamill) [inline source/_posts/2016-11-21-are-you-a-good-person.markdown:112]: `s/Sekeletor/Skeletor`

File: source/_posts/2016-11-21-are-you-a-good-person.markdown [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+I realized (with some external help as basically always) that my thinking about
+this phenomenon was all wrong. What matters is not what the show _is_, but what
+it _does_. How much feminist work does this show do? How much racist work does
+it do? These two answers can exist in parallel and allow for a much more robust
+criticism.
+
+So let's get back to the, "He may be a racist, but I'm not," thing. This
+importance of action over identity is the mechanism by which racism is
+communicable. Donald Trump does racist work. He takes actions that have racist
+outcomes. A person's support of him maginfies the effects of his work _including
```
Comment (benhamill) [inline source/_posts/2016-11-21-are-you-a-good-person.markdown:67]: `s/maginfies/magnifies`

File: source/_posts/2016-11-21-are-you-a-good-person.markdown [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+the racist work_. So supporting him is racist work. This is true regardless of
+what is in your heart or what reasoning you have.
+
+
+## You Must Observe Yourself to Discover Your Priorities
+
+One thing I've long held is that a verbal statement of priority means basically
+nothing. You don't set priorities by declaring them, you _enact_ them by _doing
+things_. Changing your habitual actions is a lot harder than saying you will
+change them and introspecting is a lot more challengeing than even that. But if
```
Comment (benhamill) [inline source/_posts/2016-11-21-are-you-a-good-person.markdown:77]: `s/challengeing/challenging`

File: source/_posts/2016-11-21-are-you-a-good-person.markdown [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+The strategy I've taken to using is that I inspect the actual actions I take in
+the world and use that to _find out_ what my priorities are. It's one thing to
+say, "School work is a high priority for me," and another to actually set aside
+the time to study and do the work. I did not, it turns out, make school work a
+high priority when I was in college.
+
+
+So when you do something that constitues racist work (and let's give you the
+benefit of the doubt that it does _other_ work that is beneficial), you are
+making a clear, unambiguous statement of priority. You are saying that the
```
Comment (benhamill) [inline source/_posts/2016-11-21-are-you-a-good-person.markdown:90]: "Statement" is confusing, here. Use "expression" or similar and reword accordingly.

File: source/_posts/2016-11-21-are-you-a-good-person.markdown [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+The problem with this thinking is a failure to understand that racism (and other
+oppressions) is catching. If you have a misogynistic friend and you don't call
+them on it, you are engaging in misogyny. If you give support, aid, succor or
+money to someone who engages in racism, you are supporting their racism. You
+cannot divest yourself of the moral burden of that fact by stepping away a
+single step (or any number of steps). The racism rubs off on you.
+
+"But wait, Ben," you might say, "Doesn't that means we're _all_ racists? This
+country ran on killing indiginous people and the slavery of black people for
+basically the first 100 years of its existance!" Here's the thing: You're right.
```
Comment (h3h) [inline source/_posts/2016-11-21-are-you-a-good-person.markdown:35]: “existence”

**Comment by benhamill** [comment]
After talking about this with a lot of folks, I think this is something I needed to write to get it out of my system, but not something I need to publish.

The main point I think I was trying to make is made better [by John Scalzi on his own blog](http://whatever.scalzi.com/2016/11/10/the-cinemax-theory-of-racism/). Professional writers making points well. It's a thing, apparently.

**Pull request closed without merging**